bin/create-scxa-analytics-schema.sh

```diff
@@ -216,39 +193,32 @@ curl -X POST -H 'Content-type:application/json' --data-binary '{
 }' http://$HOST/solr/$CORE/schema
 
 #############################################################################################
+# Fields required for BioSolr
 
-printf "\n\nDelete field conditions_search "
-curl -X POST -H 'Content-type:application/json' --data-binary '{
-  "delete-field" :
-  {
-    "name": "conditions_search"
-  }
-}' http://$HOST/solr/$CORE/schema
-
-printf "\n\nDelete field type text_en_tight "
+printf "\n\nDelete field type text_general "
 curl -X POST -H 'Content-type:application/json' --data-binary '{
   "delete-field-type":
   {
-    "name": "text_en_tight"
+    "name": "text_general"
   }
 }' http://$HOST/solr/$CORE/schema
 
-printf "\n\nCreate field type text_en_tight "
+printf "\n\nCreate field type text_general "
 curl -X POST -H 'Content-type:application/json' --data-binary '{
   "add-field-type": {
-    "name": "text_en_tight",
+    "name": "text_general",
     "class": "solr.TextField",
     "positionIncrementGap": "100",
     "analyzer" : {
       "tokenizer": {
-        "class": "solr.WhitespaceTokenizerFactory"
+        "class": "solr.StandardTokenizerFactory"
```

Comment:
Is the change of tokenizer required/suggested by BioSolr?

Reply:
I really can't remember... It's been 3 months!! I'll have to look into it.